src/components/bcros/dialog/CardedModal.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <UModal
+    :attach="attach || ''"
+    :model-value="display"
+    :data-cy="'bcros-dialog' + (name ? `-${name}` : '')"
+  >
+    <UCard
+      v-if="options"
+      :ui="{
+        header: {
+          background: 'bg-bcGovColor-darkBlue'
+        }
+      }"
+    >
+      <template #header>
+        <UIcon v-if="options.alertIcon" name="i-mdi-information-outline" class="text-4xl text-red-500 mb-2" />
+        <h2 data-cy="bcros-dialog-title" class="text-white">
```

Comment:
I would wrap `h2` into the `<slot>` so next person could reuse/change that with template if needed

Reply:
another reason for wrapping it in slot is, for example if I would pass background white for the header, I would not be able to change text color of h2.

Reply:
Good idea, as it stands you can't colour the header so I made that changeable too